src/components/Onboarding/OnboardingModal.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Modal, springs, Viewport } from '@aragon/ui'
+import { Transition } from 'react-spring/renderprops'
+import { useArrows, useSteps } from '../../hooks/useOnboarding'
+import { highlights } from './content'
+import Navigation from './Navigation'
+import HighlightScreen, { RATIO_LEFT } from './HighlightScreen'
+
+const OnboardingModal = React.memo(({ visible, onStartCourt }) => {
+  const content = highlights.latest
+  const steps = content.length
+  const { step, next, prev, setStep, direction } = useSteps(steps)
+
+  useArrows(visible ? { onLeft: prev, onRight: next } : {})
+
+  useEffect(() => {
+    if (visible) {
+      setStep(0)
+    }
+  }, [setStep, visible])
+
+  return (
+    <Viewport>
+      {({ width, height }) => {
+        const verticalMode = width < 900
+        const compactMode = width < 500 || height < 400
+        return (
+          <Modal
+            padding={0}
+            width={Math.min(1055, width - 40)}
+            visible={visible}
+            onClose={() => {}}
+            css={`
+              z-index: 4;
+            `}
+          >
+            <div
```

Comment:
Let's leave the close button (we're going to want it for the mainnet modal), but perhaps commented out for now.

Reply:
Done in https://github.com/aragon/court-dashboard/pull/76/commits/bba8b86260a81d748234fad06b02c61d2e01e724